bot/exts/fun/user_events.py

```diff
@@ -0,0 +1,583 @@
+import asyncio
+import calendar
+import logging
+from datetime import datetime, timedelta
+from functools import wraps
+from typing import Callable, Optional, Tuple
+
+from dateutil.parser import isoparse, parse
+from dateutil.relativedelta import relativedelta
+from discord import Embed, Member, Message, Reaction
+from discord.ext.commands import Cog, CommandInvokeError, Context, group, has_role
+
+from bot.api import ResponseCodeError
+from bot.bot import Bot
+from bot.constants import Channels, Guild, Roles
+from bot.utils.scheduling import Scheduler
+from bot.utils.time import humanize_delta
+
+log = logging.getLogger(__name__)
+
+EMOJIS = {
+    "check": "✅",
+    "cross": "❌"
+}
+
+DATE_PREFIX = {
+    1: 'st', 21: 'st', 31: 'st',
+    2: 'nd', 22: 'nd',
+    3: 'rd', 23: 'rd'
+}
+
+NOT_SCHEDULED = "Not scheduled"
+LIVE = "Live"
```

Comment:
Why is this a constant, if it's only used once?

Reply:
These strings had getter functions before, I asked moving them from functions to constants.